src/client/app/containers/view.mode.ts

```diff
@@ -45,6 +53,27 @@ export class ViewMode implements OnInit, OnDestroy {
         return environment.current.host.toLowerCase();
     }
 
+    get isGist() {
+        return this.snippet && this.snippet.gist;
+    }
+
+    get openInPlaygroundSupported() {
+        let host = environment.current.host.toUpperCase();
+        return Utilities.platform !== PlatformType.IOS && host === HostType.EXCEL || host === HostType.WORD || host === HostType.POWERPOINT;
```

Comment:
Please use parenthesis to make the order of precedence more clear:
Utilities.platform !== PlatformType.IOS && (host === HostType.EXCEL || host === HostType.WORD || host === HostType.POWERPOINT)

Reply:
Added.